src/pages/iou/IOUCurrencySelection.js

```diff
@@ -127,6 +145,7 @@ class IOUCurrencySelection extends Component {
                             placeholderText={this.props.translate('common.search')}
                             headerMessage={headerMessage}
                             safeAreaPaddingBottomStyle={safeAreaPaddingBottomStyle}
+                            initiallyFocusedOptionKey={_.get(_.find(this.state.currencyData, crc => crc.currencyCode === this.props.iou.selectedCurrencyCode), 'keyForList')}
```

Comment:
@iwiznia @rushatgabhane I added `initiallyFocusedOptionKey` prop to `OptionsSelector` to focus to the current currency instead of the first currency in list when we go to this page